profiles/permissions.py

```diff
@@ -20,3 +23,38 @@ def has_object_permission(self, request, view, profile):
             return True
 
         return profile.user == request.user
+
+
+class CanSeeIfNotPrivate(BasePermission):
+    """
+    Dont allow users to view private profiles.
+    Also dont allow anonymous users or non verified mm user to view profiles with
+    account_privacy is set to public public_to_mm.
+    """
+
+    def has_permission(self, request, view):
+        """
+        Only owner of a profile can view and edit their profile.
+        Verified micromaster users can view other profiles to limited view only if
+        account_privacy of profile is set to public public_to_mm
```

Comment:
this description is wrong and not about this permission class.
I do not think you need to put anything this long, given that the description is in the class docstring.

It will be enough something like "Implementation of the permission class"